dotCMS/src/main/java/com/dotmarketing/factories/PublishFactory.java

```diff
@@ -529,4 +572,26 @@ public static List getUnpublishedRelatedAssetsForPage ( IHTMLPage htmlPage, List
         return relatedAssets;
     }
 
+	private static void sendMessage(final String messageKey, String... arguments) {
+		try {
+
+			final String  message = LanguageUtil.get(messageKey, arguments);
+
+			final Role adminRole = APILocator.getRoleAPI().loadCMSAdminRole();
+			final List<String> usersId = APILocator.getRoleAPI().findUsersForRole(adminRole)
+					.stream()
+					.map(user -> user.getUserId())
+					.collect(Collectors.toList());
+
+			final SystemMessageBuilder messageBuilder = new SystemMessageBuilder()
+					.setMessage(message)
+					.setSeverity(MessageSeverity.ERROR)
+					.setType(MessageType.SIMPLE_MESSAGE)
+					.setLife(TimeUnit.SECONDS.toMillis(5));
+
+			systemMessageEventUtil.pushMessage(messageBuilder.create(), usersId);
```

Comment:
I think this notification should be sent to the user publishing the page @wezell @john-thomas-dotcms

Reply:
done https://github.com/dotCMS/core/pull/19988/commits/f79d75c145af00145cb048876e2203676ea9ebff#diff-4756d2f625956877924d160f0ccde15c4f48b7d3eb3ada335bfeb7bc993978b6R581